.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,4 @@
+command:
+  zookeeper-create-znode:
+    exec: /opt/bitnami/zookeeper/bin/zkCli.sh -server 127.0.0.1:$ZOO_PORT_NUMBER create /my_key_{{randAlpha 5}} my_data
```

Comment:
Could you please modify this test so `ZOO_PORT_NUMBER` is replaced by a vars obtained from `containerPorts.xxxxx` and hard coded in the runtime parameters? I would also try using the K8s service name here instead of `127.0.0.1` to also test that chart's resource as recommended for Spark.

About using the absolute path for the script, if I'm not mistaken it should be in the container's PATH. If we run the tests directly against `zkCli.sh -server...` we would be also checking the PATH is correctly configured.

Reply:
This has been improved